model/user/user_test.go

```diff
@@ -628,6 +631,37 @@ func (s *UserTestSuite) TestHasPermission() {
 	s.True(hasPermission)
 }
 
+func (s *UserTestSuite) TestHasDistroCreatePermission() {
+	env := &mock.Environment{}
+	s.Require().NoError(env.Configure(s.T().Context()))
```

Comment:
Small nit but do we need to configure another mock env or could we use s.env?

Reply:
It shouldn't be necessary to configure a mock env, it was just a leftover from when I realized the entire suite was using the global testing environment (not recommended since modifying that affects global state in different tests).